tests/functional/platforms/08-intelligent-host-selection/flow.cylc

```diff
@@ -0,0 +1,52 @@
+
+[meta]
```

Comment:
This should allow you to have a go at testing.
At the moment the total absence of contactable hosts will cause an exception in platforms. I imagine that if this occurs in Job submission we might want to treat that as submit-fail, and trigger retry logic.